### PR TITLE
feat(ui): More prominent adoption and stability on release detail page

### PR DESCRIPTION
**Pull request by matejminar**
image

### CODE REVIEW

File: docs-ui/components/notAvailable.stories.js [hunk 1]
```diff
@@ -25,6 +25,10 @@ export const Default = withInfo(
         </div>
       </PanelTable>
     </div>
+    <div className="section">
+      <h3>With Tooltip</h3>
+      <NotAvailable tooltip="Reason why this is not available" />
```
Comment (priscilawebdev) [inline docs-ui/components/notAvailable.stories.js:30]: good one!!

File: src/sentry/static/sentry/app/components/notAvailable.tsx [hunk 2]
```diff
@@ -1,8 +1,21 @@
 import React from 'react';
 import styled from '@emotion/styled';
 
-function NotAvailable() {
-  return <Wrapper>{'\u2014'}</Wrapper>;
+import Tooltip from 'app/components/tooltip';
+import {defined} from 'app/utils';
+
+type Props = {
+  tooltip?: string;
```
Comment (priscilawebdev) [inline src/sentry/static/sentry/app/components/notAvailable.tsx:8]: maybe we would like to pass a `tct` translation here and unfortunately, the return type is not `string`, so I think we should update it to be `React.ReactNode `

File: src/sentry/static/sentry/app/constants/notAvailableMessages.tsx [hunk 3]
```diff
@@ -0,0 +1,9 @@
+import {t} from 'app/locale';
+
+const NOT_AVAILABLE_MESSAGES = {
+  performance: t('This view is only available with Performance Monitoring.'),
```
Comment (priscilawebdev) [inline src/sentry/static/sentry/app/constants/notAvailableMessages.tsx:4]: shouldn't we be more specific here? maybe by adding the word `feature`?

**Review by priscilawebdev — COMMENTED** [review]
lgtm

**Review by priscilawebdev — APPROVED** [review]
lgtm

**Review by markstory — APPROVED** [review]
Looks good to me.

**Pull request merged**